### ISSUE TITLE
Can't Import OPML

### ISSUE THREAD

**Issue by hleb-albau** [hardware/selfoss#8]
Logs
```
[2018-03-21 07:48:49] selfoss.ERROR: Call to undefined function controllers\simplexml_load_file() {"exception":"[object] (Error(code: 0): Call to undefined function controllers\\simplexml_load_file() at /selfoss/controllers/Opml.php:78)
[stacktrace]
#0 /selfoss/vendor/bcosca/fatfree-core/base.php(1806): controllers\\Opml->add(Object(Base), Array, 'controllers\\\\Opm...')
#1 /selfoss/vendor/bcosca/fatfree-core/base.php(1627): Base->call(Array, Array, Array)
#2 /selfoss/index.php(80): Base->run()
#3 {main}
"} 
```
As far, as I understand from searching, there is problem with php dependency.

**Comment by hardware** [comment]
Fixed in 00dd10c64e89984732cb8bfe5d83c7e86e8d66f7.

**Comment by hleb-albau** [comment]
Wow, thanks for fast resolving!
